modules/log6x-logging-loki-gui-install.adoc

```diff
@@ -0,0 +1,137 @@
+// Module is included in the following assemblies:
+//
+// * observability/logging/log_storage/installing-log-storage.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-logging-loki-gui-install_{context}"]
+= Installing {logging-uc} by using the web console
+
+Install {loki-op} on your {product-title} cluster to manage the log store `Loki` from the OperatorHub by using the {product-title} web console. You can deploy and configure the `Loki` log store by reconciling the resource LokiStack with the {loki-op}.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have access to the {product-title} web console.
+* You have access to a supported object store (AWS S3, Google Cloud Storage, Azure, Swift, Minio, {rh-storage}).
+
+.Procedure
+
+. In the {product-title} web console *Administrator* perspective, go to *Operators* -> *OperatorHub*.
+
+. Type {loki-op} in the *Filter by keyword* field. Click *{loki-op}* in the list of available Operators, and then click *Install*.
++
+[IMPORTANT]
+====
+The Community {loki-op} is not supported by Red{nbsp}Hat.
+====
+
+. Select *stable-x.y* as the *Update channel*.
++
+The {loki-op} must be deployed to the global operator group namespace `openshift-operators-redhat`, so the *Installation mode* and *Installed Namespace* are already selected. If this namespace does not already exist, it will be created for you.
```

Comment:
```suggestion
The {loki-op} must be deployed to the global Operator group namespace `openshift-operators-redhat`, so the *Installation mode* and *Installed Namespace* are already selected. If this namespace does not already exist, it will be created for you.
```